### DISCUSSION TITLE
Receiving error on auth login

### DISCUSSION THREAD

**Discussion by mkjp2011** [Flagsmith/flagsmith#285] [Q&A] [answered]
When making the following curl command:

`curl.exe 'https://api.flagsmith.com/api/v1/auth/login/' -H 'content-type: application/json' --data-binary '{"name":"****","password":"****"}'`

I am getting the below error:

`{"non_field_errors":["Unable to login with provided credentials."]}`

I log in using Google account

Any idea what is going on here?

Also in the documentation, I see `{"key":"<KEY HASH>"}` being passed in. What is this?

**Comment by dabeeeenster** [comment, marked as answer]
Have you registered an account with a username and password?

**Reply by mkjp2011** [reply]
When I signed up I used the Google button functionality. Do I need to sign up without using that?

**Reply by dabeeeenster** [reply]
Yes you would need to provide a password

**Reply by mkjp2011** [reply]
That did it. To fix this, I had to reset my password to establish one! Thank you!
